src/tests/searchParameters/searchByAddress/fixtures/Person/person1.json

```diff
@@ -0,0 +1,71 @@
+{
+    "resourceType": "Person",
+    "id": "1",
+    "meta": {
+        "versionId": "1",
+        "lastUpdated": "2022-06-13T13:52:25.000Z",
+        "security": [
+            {
+                "system": "https://www.icanbwell.com/access",
+                "code": "unitypoint"
+            },
```

Comment:
1. format all json files with 2 space indentation
2. Can we create a single bundle for all the patients?